daemon/src/config.rs

```diff
@@ -131,6 +131,16 @@ pub fn has_git_remote(path: &Path) -> bool {
     false
 }
 
+pub fn ethersync_directory_is_ignored(path: &Path) -> bool {
+    if let Ok(repo) = git2::Repository::discover(path) {
+        let ethersync_dir = path.join(CONFIG_DIR);
+        return repo
+            .is_path_ignored(ethersync_dir)
+            .expect("Should have been able to determine ignore state for {path}");
+    }
+    false
```

Comment:
Another thing: If there's no Git repo, I'd say no action is needed. So either rename the function to something like `..._should_be_ignored_but_isnt` or call it in a context when we've detected a Git repo?